CHANGELOG.md

```diff
@@ -68,6 +69,38 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.3.0
+
+### Change
+
+- Change the default router from `radixtree_uri` to `radixtree_host_uri`: [#9047](https://github.com/apache/apisix/pull/9047)
```

Comment:
this is a break change

Reply:
Would it be better to add an underline or note ?